examples/bri-3/src/bri/workgroup/workgroups/api/workgroups.controller.ts

```diff
@@ -28,14 +29,14 @@ export class WorkgroupController {
   @Post()
   async createWorkgroup(
     @Body() requestDto: CreateWorkgroupDto,
+    @Headers('authorization') accessToken: string, //TODO replace with JWT type from its module
   ): Promise<string> {
     return await this.commandBus.execute(
       new CreateWorkgroupCommand(
+        accessToken,
```

Comment:
I created a custom decorator in the shared folder containing the public endpoint decorator that takes in the jwt from the header, decodes it and passes back the pubkey.

Reply:
Yes, passing access token to every command that needs to be aware of the currently logged in user is probably not the optimal option. We could use something like this https://www.npmjs.com/package/request-context and then have a wrapper around it so that we can fetch the bpi subject id wherever needed. Would you mind adding a TODO and opening an issue to discuss further?